src/shared/services/map-location/index.js

```diff
@@ -56,4 +73,36 @@ export const formatAddress = address => {
   return display;
 };
 
+export const serviceResult2Address = ({ straatnaam_verkort, huis_nlt, postcode, woonplaatsnaam }) => ({
```

Comment:
I just noticed this; why not write out the words, like `serviceResultToAddress`? Why the `2`?

Reply:
No particular reason. I'll use `To` from now on.